pyvista/plotting/plotter.py

```diff
@@ -2918,13 +2921,10 @@ def add_mesh(
             updated.  If an actor of this name already exists in the
             rendering window, it will be replaced by the new actor.
 
-        texture : vtk.vtkTexture or np.ndarray or bool or str, optional
+        texture : vtk.vtkTexture or np.ndarray, optional
             A texture to apply if the input mesh has texture
             coordinates.  This will not work with MultiBlock
-            datasets. If set to ``True``, the first available texture
-            on the object will be used. If a string name is given, it
-            will pull a texture with that name associated to the input
-            mesh.
+            datasets.
```

Comment:
I'm surprised this didn't work for `MultiBlock` in case the texture was encoded in the mesh. Was it really unsupported? Because if it worked after all then this will be a breaking change once we remove the feature.

Reply:
> Was it really unsupported?

I'm not sure. I noticed it in the docstring and had the thought that you really shouldn't do texture plotting this way with multiblock datasets so I stripped this out.

> Because if it worked after all then this will be a breaking change once we remove the feature.

I'm not sure I follow?

Reply:
My thinking is that if meshes track their textures, and multiblocks track their constituent meshes, then `texture=True` might work if every mesh of the multiblock has an associated texture, and the result would be intuitive. If we then deprecate and later remove the option to have associated textures on meshes, we'll lose this feature of being able to plot multiblocks with textures, because you can't pass an external texture that makes sense for the whole multiblock.

Is this clearer?

Reply:
Ah, yes, very clear. Indeed this would be breaking in that way. If I had to guess, there may be one single person out there using the API in this way... if at all, so a break I'm willing to let slide